proposals/privacy.md

```diff
@@ -132,14 +136,18 @@ links (identified by relation type);
 metadata (creation time, modification time, support, and version information);
 data schemas;
 and URI templates.
+ML: This mechanism is not part of the current specs.
 
 ### TDs MUST be protected by encryption when at rest.
 TDs must be stored in an encrypted data store, and access to this store should be limited to authorized users.
+ML: This goes a bit far - We should constrain it to "TDs containing PII"
```

Comment:
This is constrained to "TDs associated with an identifiable person", from which PII can be inferred.